README.md

```diff
@@ -80,6 +80,21 @@ async function showPerson(person) {
 }
 ```
 
+
+### Get all subjects from a document
+```javascript
+const typeIndex = path.create({ subject: namedNode('https://jshurmer.inrupt.net/settings/publicTypeIndex.ttl') });
+showTypes(typeIndex);
```

Comment:
Maybe we can have a different example? The subjects/types link might be confusing.